openpype/modules/kitsu/plugins/publish/integrate_kitsu_review.py

```diff
@@ -29,10 +30,46 @@ def process(self, instance):
             # Skip if not tagged as review
             if "kitsureview" not in representation.get("tags", []):
                 continue
+
+            filenames = representation.get("files")
+
+            frame_range = instance.data.get("customFrames", [])
```

Comment:
Same comment as for the `openpype/plugins/publish/integrate.py` file, the name isn 't reflecting the content of the variable

Reply:
j'ai renommé frame_range en custom_frames